pvlib/irradiance.py

```diff
@@ -2151,7 +2151,12 @@ def _dirint_bins(times, kt_prime, zenith, w, delta_kt_prime):
     return kt_prime_bin, zenith_bin, w_bin, delta_kt_prime_bin
 
 
-def dirindex(ghi, ghi_clearsky, dni_clearsky, zenith, times, pressure=101325.,
+@renamed_kwarg_warning(
+    since='0.11.2',
+    old_param_name='dni_clearsky',
+    new_param_name='dni_clear',
+    removal="0.12.0")
```

Comment:
I'm hesitant to say we'll remove the deprecations this soon.  0.12.0 could potentially be in the next couple releases, which means the deprecation period would only be 3 or 6 months.  I don't think pvlib has a firm convention on this, but I remember saying "at least 12 months" previously.  

Input from others would be helpful here.

Reply:
I think there is no harm in supporting old names for 12 months. +1 for `v0.13.0`.